db-migration/Cargo.toml

```diff
@@ -15,6 +15,9 @@ ckb-db = { path = "../db", version = "= 0.113.0-pre" }
 ckb-logger = { path = "../util/logger", version = "= 0.113.0-pre" }
 ckb-error = { path = "../error", version = "= 0.113.0-pre" }
 ckb-db-schema = { path = "../db-schema", version = "= 0.113.0-pre" }
+ckb-channel = { path = "../util/channel", version = "= 0.113.0-pre" }
+ckb-stop-handler = { path = "../util/stop-handler", version = "= 0.113.0-pre" }
+once_cell = "1.8.0"
```

Comment:
If we can use [std::cell::OnceCell](https://doc.rust-lang.org/std/cell/struct.OnceCell.html), then we can remove `once_cell`